packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1453,6 +1455,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private addMetadataToSummary(summaryTree: ISummaryTreeWithStats) {
         const metadata: IContainerRuntimeMetadata = {
             ...this.createContainerMetadata,
+            // back-compat 0.59.3000: This is renamed to summaryNumber. Can be removed when 0.59.3000 saturates.
+            summaryCount: this.nextSummaryNumber,
```

Comment:
It looks like `summaryCount` is not gonna be the same value as `summaryNumber` as 
- `summaryCount = this.nextSummaryNumber`
- `summaryNumber = this.nextSummaryNumber++`

Reply:
Why is it the same? Is it because ++ executes after the assignment?

Reply:
Yes, post decrement operator uses the value and then increments it.